doc/conf.py

```diff
@@ -49,9 +49,9 @@
 # built documents.
 #
 # The short X.Y version.
-version = '0.6.1'
+version = '0.6.2'
```

Comment:
The version pattern don't correspond to the comment ...

Reply:
It's ok I think. `version` is the actual the version number, while `release` is the version number + the alpha/beta/rc tag. In our case `version` and `release` are the same, and we use three-digit version numbers.

Reply:
As you want but it doesn't respect the comment :-)

Reply:
Yes, that's a bit error-prone. I'll change it.

Reply:
:-)